UnicornRender/utility/sources/asset/SimpleStorage.cpp

```diff
@@ -23,25 +23,25 @@ Handler SimpleStorage::Get(const KeyType& key)
 
     if (m_entries.cend() != it)
     {
-        LOG_DEBUG("Requested asset \"%s\" -> exists", key.c_str());
+        spdlog::get(utility::Logger::LoggerName())->debug("Requested asset \"%s\" -> exists", key.c_str());
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.